pybop/optimisation.py

```diff
@@ -406,3 +410,8 @@ def set_max_unchanged_iterations(self, iterations=200, threshold=1e-11):
 
         self._unchanged_max_iterations = iterations
         self._unchanged_threshold = threshold
+
+    def store_optimised_parameters(self, x):
+        # Add the initial values to the parameter definitions
```

Comment:
Update descriptive comment

Also, what do you think about storing an `initial_value`, as well as a `value` for each parameter?
At the start of an optimisation, `value` could be copied to `initial_value` and then it would be possible for users to compare initial and optimised parameters values at the end.